### ISSUE TITLE
highest combination paramete

### ISSUE THREAD

**Issue by tghrdwadhaihfoiw** [shenmi175/TianChi-Memory-failure-prediction#1]
Hello, may I ask for your highest combination parameter of 29.37 ？

**Comment by shenmi175** [comment]
是的。由于数据量不多，在随机采样时会有很大的偏差，但均分是在25以上，我们最高做到了29.37，如果你运气足够好，你可以得到更高的分数。
[Yes. Due to the small amount of data, there will be a large deviation in random sampling, but the average score is above 25, and the highest we achieved is 29.37. If you are lucky enough, you can get a higher score.]

**Comment by tghrdwadhaihfoiw** [comment]
thanks



---Original---
From: ***@***.***&gt;
Date: Mon, Dec 23, 2024 09:54 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [shenmi175/TianChi-Memory-failure-prediction] highest combinationparamete (Issue #1)




 
是的。由于数据量不多，在随机采样时会有很大的偏差，但均分是在25以上，我们最高做到了29.37，如果你运气足够好，你可以得到更高的分数。
 [Yes. Due to the small amount of data, there will be a large deviation in random sampling, but the average score is above 25, and the highest we achieved is 29.37. If you are lucky enough, you can get a higher score.]
